api/spec/v1.yaml

```diff
@@ -1511,6 +1511,9 @@ components:
           type: string
           description: The delegatee (validator) address.
           example: *staking_address_1
+        validator_name:
```

Comment:
The API's approach to handling validator metadata has become somewhat inconsistent.

Currently, we return `validator_name` for this endpoints, `EntityInfo` for block proposers and signers, and `ValidatorMedia` for validator-specific endpoints. Although these represent the same underlying data, the choice of response fields across different endpoints feels somewhat arbitrary.

To improve this, I suggest either:

1) Unify the responses so all relevant endpoints return the same metadata structure, or
2) (my preference, but apparently not the forntend's): Keep metadata to be returned only by the validator endpoint. The frontend can cache and reuse this information across different parts of the app.

---

Main Reasoning for Option 2:
- If a delegator is a validator, the frontend would likely want to show the validator name. With the current approach, we'd need to introduce additional fields like `delegator_name` to the delegation endpoint
- Consider other endpoints, like `consensus/account/{address}`. When querying account information for a validator, the frontend would want to show the validator's name. Following the current setup, this would mean adding `ValidatorMedia` to the account endpoint response.
- Similarly, if a consensus transaction was submitted by an address that belongs to a validator (or if the recipient is a validator), the frontend would likely want to display the validator's name. This would mean adding the validator metadata to yet another endpoint, resulting in more redundancy.

Reply:
> (my preference, but apparently not the forntend's): Keep metadata to be returned only by the validator endpoint. 
We have discussed this in https://app.clickup.com/t/869683ayv and I was under impression that we agreed to do it on frontend side (PR was created last week https://github.com/oasisprotocol/explorer/pull/1576) because we need names in more views.